semcore/data-table/src/Head.tsx

```diff
@@ -48,10 +52,86 @@ class Head extends Component<AsProps> {
     }
   };
 
-  refColumn = (props: Column['props']) => (ref: HTMLElement) => {
-    setRef(props.ref, ref);
-    if (props.forwardRef) {
-      setRef(props.forwardRef, ref);
+  refSort = (active: boolean) => (ref: HTMLElement | null) => {
```

Comment:
It it ok that function doesn't consider what column it relates to? Afaik we may have more than one sortable column

Reply:
As I see - no, we could have only one sortable column in a time